src/components/Camera/index.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { TouchableOpacity, View } from "react-native";
+import Icon from "react-native-vector-icons/FontAwesome";
+import { RNCamera } from "react-native-camera";
+
+import styles from "./styles";
+
+const PICTURE_OPTIONS = {
+  quality: 1,
+  fixOrientation: true,
+  forceUpOrientation: true
+};
+
+export default class App extends React.Component {
+  state = {};
+
+  flip = () => {
+    this.setState({
+      type:
+        this.state.type === RNCamera.Constants.Type.back
```

Comment:
https://medium.com/@WoTzap/%D1%84%D1%83%D0%BD%D0%BA%D1%86%D0%B8%D0%BE%D0%BD%D0%B0%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-setstate-%D1%8D%D1%82%D0%BE-%D0%B1%D1%83%D0%B4%D1%83%D1%89%D0%B5%D0%B5-react-d262be6a6c02
functional setState is preferred but not required